src/components/updatemenus/defaults.js

```diff
@@ -48,7 +50,9 @@ function menuDefaults(menuIn, menuOut, layoutOut) {
     var visible = coerce('visible', buttons.length > 0);
     if(!visible) return;
 
-    coerce('active');
+    // Default to zero since active must be *something*:
```

Comment:
Ah, good point. That works for me. I think that's best since `undefined` isn't a state that really makes sense (because it's always *something*).